lib/main_containers.pm

```diff
@@ -131,12 +139,13 @@ sub load_host_tests_containerd_nerdctl {
     loadtest 'containers/containerd_nerdctl';
 }
 
-sub load_host_tests_helm() {
+sub load_host_tests_helm {
+    my ($run_args) = @_;
     if (is_sle('15-sp3+')) {
-        loadtest "containers/helm_eks";
-        loadtest "containers/helm_k3s";
+        load_helm_test('eks');
+        load_helm_test('k3s');
```

Comment:
Please follow the rest of the file and do not redefine the `$run_args`.
```suggestion
        $run_args->{k8s_env} = 'eks';
        load_helm_test($run_args);
        $run_args->{k8s_env} = 'k3s';
        load_helm_test($run_args);
```

Reply:
New approach